ttnn/cpp/ttnn/operations/eltwise/unary/unary_composite.hpp

```diff
@@ -97,67 +97,53 @@ struct ExecuteUnaryCompositeOp
-struct ExecuteUnaryCompositeOpWithScaleShift
-{
-    static ttnn::Tensor execute_on_worker_thread(
-        const Tensor& input_tensor,
-        float scale,
-        float shift,
-        const std::optional<MemoryConfig>& memory_config = std::nullopt)
-        {
-            auto op_type = get_function_type2<unary_comp_op_type>();
-            return op_type(input_tensor, scale, shift, memory_config);
+struct ExecuteUnaryCompositeOpWithTwoFloat {
+
+    //Type 1: 1 inputs, 2 float
+    static ttnn::Tensor execute_on_main_thread(
+        const Tensor &input_tensor,
+        float param1,
+        float param2,
+        const std::optional<MemoryConfig> &memory_config = std::nullopt) {
+        auto op_type = get_function_type_two_float<unary_comp_op_type>();
+        auto output_memory_config = memory_config.value_or(input_tensor.memory_config());
+        return op_type(input_tensor, param1, param2, output_memory_config);
         }
-
 };
 
-template <UnaryCompositeOpType unary_comp_op_type>
-struct ExecuteUnaryCompositeOpWithLowHigh
-{
-    static ttnn::Tensor execute_on_worker_thread(
-        const Tensor& input_tensor,
-        float low,
-        float high,
-        const std::optional<MemoryConfig>& memory_config = std::nullopt)
-        {
-            auto op_type = get_function_type3<unary_comp_op_type>();
-            return op_type(input_tensor, low, high, memory_config);
-        }
-
-};
 
+//OpHandler_one_int : get_function_type_one_int
 template <UnaryCompositeOpType unary_comp_op_type>
-struct ExecuteUnaryCompositeOpWithScaleAlpha
-{
-    static ttnn::Tensor execute_on_worker_thread(
-        const Tensor& input_tensor,
-        float scale,
-        float alpha,
-        const std::optional<MemoryConfig>& memory_config = std::nullopt)
-        {
-            auto op_type = get_function_type4<unary_comp_op_type>();
-            return op_type(input_tensor, scale, alpha, memory_config);
+struct ExecuteUnaryCompositeOpWithOneInt {
+
+    //Type 1: 1 inputs, 2 float
+    static ttnn::Tensor execute_on_main_thread(
+        const Tensor &input_tensor,
+        int32_t param1,
+        const std::optional<MemoryConfig> &memory_config = std::nullopt) {
+        auto op_type = get_function_type_one_int<unary_comp_op_type>();
+        auto output_memory_config = memory_config.value_or(input_tensor.memory_config());
+        return op_type(input_tensor, param1, output_memory_config);
         }
-
 };
 
 template <UnaryCompositeOpType unary_comp_op_type>
-struct ExecuteUnaryCompositeOpWithThresholdValue
-{
-    static ttnn::Tensor execute_on_worker_thread(
-        const Tensor& input_tensor,
-        float threshold,
-        float value,
-        const std::optional<MemoryConfig>& memory_config = std::nullopt)
-        {
-            auto op_type = get_function_type5<unary_comp_op_type>();
-            return op_type(input_tensor, threshold, value, memory_config);
+struct ExecuteUnaryCompositeOpWithOneIntBool {
```

Comment:
if they are ok with this name, go ahead.

Reply:
I am not ok with this sort of name.   Having type_1, or "OpWithOneIntBool" does not communicate the feature intent of the op.   This at least indicates ExecuteUnaryCompositeOpWithThresholdValue